### ISSUE TITLE
[Windows Build] STOTIKS, please enlighten us with your vast knowledge 

### ISSUE THREAD

**Issue by Taroc54** [madMAx43v3r/chia-plotter#397]
I'm interested in building from scratch.

I cloned your master branch. Used cygwin to make the build, installed all the required dependencies. 
I created the build successfully with no errors but the chia-plotter.exe doesn't work. I put in the plotting cmd nothing happens and no errors.

Please, will you be so kind to walk us through how you created the build.  😀.

I can't create an issue on your branch so I had to make it here.
@stotiks

**Comment by gilmotta** [comment]
Yes, I also placed a request to be able to build for Windows. I don't think that the Windows build from Stotiks is to be used with Cygwin otherwise the binary would require the cygwin.dll and I have not seen that dependency. I think that he used Visual Studio but it is my BIG guess. I started the port to Visual Studio and I should be done in a week or so depending on how busy I am at my job.

If Stotiks can share his instructions with us, it will save us some time.
